### DISCUSSION TITLE
App Catalog returning 0 results

### DISCUSSION THREAD

**Discussion by mschuess** [microsoft/powerapps-tools#957] [General]
I recently installed the Core Components of the CoE. I noticed that the App Catalog Canvas App is not returning any results. Do you know what might be the problem? 

![image](https://user-images.githubusercontent.com/5606752/115758621-efca4600-a364-11eb-8595-9434648e2e6c.png)

**Comment by manuelap-msft** [comment]
Only apps that an admin marks as "highlight in app catalog" are shown in the App Catalog. The process is described here https://docs.microsoft.com/en-us/power-platform/guidance/coe/example-processes
